terraform/constellation-cluster/install-constellation.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+if [[ -f ./constellation ]]; then
+  echo "constellation CLI is already available."
+  exit 0
+fi
+
+echo "Fetching constellation ${VERSION}"
+OS=$(uname -s)
+ARCH=$(uname -m)
+VERSION="latest"
```

Comment:
I'd like to discuss what version should be downloaded [again](https://github.com/edgelesssys/constellation/pull/2503#discussion_r1370296925). Is it possible to pass a version value from Terraform to this script?

If we go with latest and one downloads this TF module, it could break if we release a new version that requires tf changes, right?
It also wouldn't be possible to stay with an older release.

Reply:
The script would break, if you download the module, don't use it and meanwhile another CLI is released. I will expose the cli version as a TF variable.

Reply:
So do you think it's necessary to inject the CLI version during the release process?